app/helpers/title_helper.rb

```diff
@@ -0,0 +1,266 @@
+# frozen_string_literal: true
+
+# --------- Contextual Page Title -----------------------------
+#
+#  add_page_title(title)        # add content_for(:title)
+#                                 and content_for(:document_title)
+#  add_owner_naming(naming)     # add content_for(:owner_naming), on show obs
+#  title_tag_contents           # text to put in html header <title>
+#  add_index_title              # logic for index titles, with fallbacks
+#  index_default_title          # logic for observations index default sort
+#  add_query_filters(query)     # content_for(:filters)
+#                                 builds filter caption explaining
+#                                 index results, if filtered
+#
+module TitleHelper
+  # sets both the html doc title and the title for the page (previously @title)
+  def add_page_title(title)
+    content_for(:title) do
+      title
+    end
+    content_for(:document_title) do
+      title_tag_contents(title)
+    end
+  end
+
+  # Show obs: observer's preferred naming. HTML here in case there is no naming
+  def add_owner_naming(naming)
+    return unless naming
+
+    content_for(:owner_naming) do
+      tag.h5(naming, id: "owner_naming")
+    end
+  end
+
+  # contents of the <title> in html <head>
+  def title_tag_contents(title, action: controller.action_name)
+    if title.present?
+      title.strip_html.unescape_html # removes tags and special chars
+    elsif TranslationString.where(tag: "title_for_#{action}").present?
+      :"title_for_#{action}".t
```

Comment:
Not covered.

Reply:
Ah! Thanks

Reply:
This was dead code. Removed, along with translation strings, a fixture and tests.